index.html

```diff
@@ -802,9 +849,34 @@
     {name: "java"},
     {name: "cpp"},
     {name: "python"},
-    {name: "go", tabSize: 4, indentWithTabs: true, indentUnit: 4}
+    {name: "go", tabSize: 2, indentWithTabs: true, indentUnit: 4}
   ].forEach(spec => {
     editors[spec.name] = CodeMirror.fromTextArea(document.querySelector(`#code-${spec.name}`),
-                                                 Object.assign({mode: "google-" + spec.name}, spec))
+   Object.assign({
+    mode: "google-" + spec.name,
+    autoCloseBrackets: true,
+    continueComments: {key: "Enter", continueLineComment: false},
+    foldGutter: true,
+    gutters: ["gutter", "CodeMirror-linenumbers", "CodeMirror-foldgutter"],
+    height: "100%",
+    highlightSelectionMatches: {showToken: true, delay: 500, annotateScrollbar: true},
+    indentUnit: 2,
+    inputStyle: "textarea",
+    lineNumbers: true,
+    lineWrapping: false,
+    matchBrackets: {maxScanLines: 5000},
+    pasteLinesPerSelection: false,
+    readOnly: false,
+    scrollPastEnd: false,
+    showCursorWhenSelecting: true,
+    showTrailingSpace: true,
+    smartIndent: true,
+    statementIndentUnit: 4,
```

Comment:
This isn't an option that exists (it exists as a mode option in the old clike mode, but definitely not as an editor option)

Reply:
Removed.